README.md

```diff
@@ -40,6 +40,7 @@ To run the app, follow these steps.
   gulp watch
   ```
 7. Browse to [http://localhost:9000](http://localhost:9000) to see the app. You can make changes in the code found under `src` and the browser should auto-refresh itself as you save files.
+> The Skeleton App uses [BrowserSync](http://www.browsersync.io/) for automated page refreshes on code/markup changes concurrently accross multiple browsers. If you prefer to disable the mirroring feature set the [ghostMode option](http://www.browsersync.io/docs/options/#option-ghostMode) to false
```

Comment:
I think this should be a little more explicit about mirroring. While I found the automated refreshes a very expectable behaviour, "mirroring" key presses across all browsing devices is something quite unexpected. Maybe I would add a sentence in between this two, to explain what mirroring means here: `It can also mirror clicks, scrolls & form inputs across all connected devices.`